draft-ietf-ppm-dap.md

```diff
@@ -1377,21 +1383,26 @@ This message consists of:
 * `prepare_inits`: the sequence of `PrepareInit` messages constructed in the
   previous step.
 
-Finally, the Leader sends a PUT request to
-`{helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}`. The payload
-is set to `AggregationJobInitReq` and the media type is set to
+Finally, the Leader sends a request to
+`PUT {helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}`. The
+payload is set to `AggregationJobInitReq` and the media type is set to
 "application/dap-aggregation-job-init-req".
 
 The Leader MUST authenticate its requests to the Helper using a scheme that
 meets the requirements in {{request-authentication}}.
 
+After receiving HTTP status 201 Created, the Leader proceeds to poll the
+aggregation job by sending requests to
+`GET {helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}?step=0`
+until receiving HTTP status 200 OK with a body containing an
+`AggregationJobResp`. The Helper's response SHOULD include a Retry-After header
+field to suggest a polling interval to the Leader.
```

Comment:
This last sentence suggests that the eventual 200 OK `AggregationJobResp` response should come with a `Retry-After` header. I think we should reorder the sentences.